sled-agent/src/illumos/zpool.rs

```diff
@@ -154,6 +162,21 @@ pub struct Zpool {}
 
 #[cfg_attr(test, mockall::automock)]
 impl Zpool {
+    pub fn create(
+        name: ZpoolName,
+        vdev: &std::path::PathBuf,
+    ) -> Result<(), CreateError> {
+        let mut command = std::process::Command::new(PFEXEC);
+        let cmd = command.args(&[
+            ZPOOL,
+            "create",
+            &name.to_string(),
+            &vdev.to_string_lossy(),
```

Comment:
I think it's probably best to always use `.arg()`, generally, so that you don't have to worry about string type specifics, and you can group related arguments; i.e.,

```rust
let mut cmd = std::process::Command::new(PFEXEC);
cmd.arg(ZPOOL).arg("create");
cmd.arg(&name.to_string()); /* does this not implement an .as_str() or Display? */
cmd.arg(vdev);
```

Reply:
Ah, thanks, calling `arg` repeatedly does make the conversion a lot easier. Updated.